.github/workflows/workspace.yml

```diff
@@ -26,7 +27,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout sources
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
+
+      - uses: RustCrypto/actions/cargo-cache@master
```

Comment:
I don't think the caching is needed for `rustfmt`.

Reply:
True, rustfmt doesn't  update the crates.io index. I updated the commit.